backend/src/main/java/com/jobwait/fakedata/NormalRandom.java

```diff
@@ -0,0 +1,11 @@
+package com.jobwait.fakedata;
+
+import java.util.Random;
+
+public class NormalRandom {
+    private static Random random = new Random();
+
+    static int sampleInt(int average, int std) {
+        return (int) (random.nextGaussian() * std + average);
+    }
```

Comment:
now this is stats

Reply:
We are swiftly approaching the need for a statistics library (Spark moment) but I think we can scrape right up to the edge  of needing it and still finish the project without having to go sicko mode on the stats